libs/prelude/Prelude.idr

```diff
@@ -37,6 +37,8 @@ import public Language.Reflection.Errors
 %default total
 
 -- Things that can't be elsewhere for import cycle reasons
+%runElab (defineFunction $ DefineFun `{void} [])
```

Comment:
I'm not a fan of the definition being so far from the declaration - why does it need to be that way?

I can see that Builtins exists prior to the existence of Language.Reflection.Elab, which means it can't be there, but what about moving the declaration of Void to Prelude.idr?